platform/ios/PolyPodApp/PolyPod/HomeScreenView.swift

```diff
@@ -409,6 +415,21 @@ struct ToolsSectionView: View {
     }
 }
 
+struct DeveloperSectionView: View {
```

Comment:
Could have used `TitleSectionView` instead of literally copying it. Figured the latter makes a wee bit more sense, but happy to change that.

Reply:
Yeah, I think it can be refactored to have more shared code between sections, if you are up to it :). At the time when HomeScreen was created, we have decided to keep sections separated in case of any major UI change.

Reply:
Seems best to keep it seperate for now :P